### PR TITLE
[delivery][docs] Add Russian translation of WerfSource CR

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

Translated into Russian the page with the description of Custom Resource WerfSource

## Why do we need it, and what problem does it solve?

At the moment, the page has not been translated.

## Why do we need it in the patch release (if we do)?

To support documentation in two languages.

## What is the expected result?

Documentation will become more convenient for native speakers of the added language.

## Checklist
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Documentation updates of the delivery module.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: ee/modules/502-delivery/crds/doc-ru-werfsource.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                  description: |
+                    Необходимость создания репозитория Argo CD для этого источника.
+
+                    Если установлено `true` — в Argo CD будет создан OCI-репозиторий c именем `name`. Отключите этот параметр, если хотите создать репозиторий вручную,
+                    например, на основе Git или Helm Chart Museum.
+                argocdRepo:
+                  description: |
+                    Конфигурация репозитория Argo CD.
+
+                    Если `argocdRepoEnabled` установлен в `true`, то в Argo CD будет создан репозиторий с именем, аналогичным имени ресурса.
```
Comment (VV-corrector) [inline ee/modules/502-delivery/crds/doc-ru-werfsource.yaml:37]:
```suggestion
                    Если `argocdRepoEnabled` установлен в `true`, в Argo CD будет создан репозиторий с именем, аналогичным имени ресурса.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: ee/modules/502-delivery/crds/doc-ru-werfsource.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+            Используется для создания конфигурации container registry для Argo CD Image Updater и в случае необходимости для создания OCI Helm-репозитория в Argo CD для использования с werf bundles.
+          properties:
+            spec:
+              properties:
+                imageRepo:
+                  description: Registry с образами.
+                apiURL:
+                  description: |
+                    Ссылка на API registry.
```
Comment (z9r5) [inline ee/modules/502-delivery/crds/doc-ru-werfsource.yaml:17]:
```suggestion
                    Адрес API registry.
```

File: ee/modules/502-delivery/crds/doc-ru-werfsource.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+                    Если не указана — автоматически берется из `imageRepo`.
```
Comment (z9r5) [inline ee/modules/502-delivery/crds/doc-ru-werfsource.yaml:19]:
```suggestion
                    Если не указана — автоматически берется из параметра [imageRepo](#werfsource-v1alpha1-spec-imagerepo).
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6032](https://github.com/deckhouse/deckhouse/pull/6032) to the branch [release-1.52](https://github.com/deckhouse/deckhouse/tree/release-1.52) successful!

**Pull request merged**